webpack/stories/docs/slot-and-fill.stories.mdx

```diff
@@ -150,10 +152,10 @@ addGlobalFill(
 );
 ```
 
-Register `fills` global file in `plugn.rb` file:
+Register `fills` global file in `plugin.rb` file:
 
 ```ruby
-Foreman::Plugin.register :<plugin> do
+Foreman::Plugin.register :plugin do
```

Comment:
Why this change? This makes it look like you always need to use `:plugin` while it's really dependent on what you use.

Reply:
@ekohl I think that's left over from when I was troubleshooting the [weirdness](https://github.com/theforeman/foreman/pull/7990#issuecomment-700238606) with mdx-loader trying to parse the same file twice.  I will change it back.